[TOOLS-4458] Create a Cubrid plugin based on generic

longhaseng52 opened the pull request:
http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin based on generic plugin

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin based on generic plugin

longhaseng52 committed:
Merge pull request #2 from longhaseng52/TOOLS-4458

[TOOLS-4458] Create a Cubrid plugin based on generic

kisoo-han commented on plugins/org.jkiss.dbeaver.ext.cubrid/OSGI-INF/l10n/bundle.properties:
```diff
@@ -0,0 +1,105 @@
+meta.org.jkiss.dbeaver.ext.cubrid.model.GenericCatalog.name.name                                = Catalog Name
```
For the naming issue (Genetic -> CUBRID),
I think you change database name from Generic to CUBRID (**database name**) over cubrid plugin, for example,

meta.org.jkiss.dbeaver.ext.cubrid.model.**Generic**Catalog.name.name
meta.org.jkiss.dbeaver.ext.cubrid.model.**CUBRID**Catalog.name.name

This could be time-consuming process, but I think it is required to change.
Please refer naming of other databases.

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- Refactoring file name

longhaseng52 committed:
Merge pull request #3 from longhaseng52/TOOLS-4458

[TOOLS-4458] Create a Cubrid plugin based on generic

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin based on generic
git commit -F- <<EOF
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- refactoring cubrid plugin based on generic
- refactoring cubrid ui plugin

longhaseng52 committed:
Merge pull request #4 from longhaseng52/TOOLS-4458

[TOOLS-4458] Create a Cubrid plugin based on generic

longhaseng52 committed:
Revert "[TOOLS-4458] Create a Cubrid plugin based on generic"

longhaseng52 committed:
Merge pull request #5 from longhaseng52/revert-4-TOOLS-4458

Revert "[TOOLS-4458] Create a Cubrid plugin based on generic"

longhaseng52 committed:
[TOOLS-4458] Create a Cubrid plugin based on generic

http://jira.cubrid.org/browse/TOOLS-4458

- register new added plugin

longhaseng52 committed:
Merge branch 'devel' into TOOLS-4458
